### ISSUE TITLE
How can be downloaded the latest version?

### ISSUE THREAD

**Issue by johnfound** [resilar/sqleet#27]
I've discovered that on https://github.com/resilar/sqleet/releases/latest is a page about the latest version of SQLeet, but is there an URL to download directly the amalgamation source of the latest version?

**Comment by resilar** [comment]
Direct link to the latest amalgamation `.zip`/`.tar.gz` package of sqleet? No, I don't think GitHub supports that. I agree that this could be an useful feature and easy to implement using HTTP redirects. I'm not aware of any URL shortening/redirection service that is free and does not suck, so implementation is not going to happen anytime soon**¹**.

Do you have a specific use case in mind? 

---

(1) sqleet is steadily reaching a state where a major version number increment to `v1.x.y` is justified in terms of stability and features. I'm considering building a (minimal) web page for sqleet and releasing it at the same time with version `1.x.y`. This obviously then allows providing a direct link to the amalgamation packages.

**Comment by johnfound** [comment]
Well, I have created some kind of workaround. The idea is to obtain the latest amalgamation source:

```
  echo "Downloading latest SQLeet master branch archive..."
  { wget -q -O sqleet.zip https://github.com/resilar/sqleet/archive/master.zip; unzip -q sqleet.zip; rm sqleet.zip; } || { echo >&2 "Error: Can't download SQLeet sources."; exit 2;}
  cd ./sqleet-master
  echo "Building the amalgamation sqlite3.c ..."
  script/amalgamate.sh < ./sqleet.c > ../sqlite.c
  cd ..
  rm -rf ./sqleet-master/ &
```
Hope this can be useful for someone.

**Comment by resilar** [comment]
Yeah, the lack of direct links is easy to overcome with shell scripting. Given git and a few extra commands, it is doable to fetch & build the latest release's amalgamation based on git tags.

Offering static links to the latest release amalgamations is still a worthwhile goal, so let's keep this issue open.
